modules/create_timepoint/js/.gitignore

```diff
@@ -0,0 +1 @@
+create_timepoint.js
```

Comment:
needs updating. i think doing *.js as suggested by @driusan in a previous PR is a good idea